### PR TITLE
Fix check

### PR DESCRIPTION
**Pull request by jreidinger**
even if previous comment that it is too dependent in future, I think customer interested is more important and when we provide "Test" functionality it should work reliable.

### CODE REVIEW

**Comment by mvidner** [comment]
Unrelated to this fix, but `LANG=en_US rspec -f d test/ntp_client_test.rb` fails with "uninitialized constant FileUtils", please `require "fileutils"`.

File: test/ntp_client_test.rb [hunk 1]
```diff
@@ -442,6 +442,17 @@
           expect(subject.sntp_test(server)).to eql(false)
         end
 
+        it "returns false if sntp respong no UCST" do
+          output["stdout"] = "sntp 4.2.8p8@1.3265-o Fri Sep 30 15:52:10 UTC 2016 (1)\n" \
+            "195.113.144.2 no UCST response after 5 seconds\n"
```
Comment (mvidner) [inline test/ntp_client_test.rb]: `output["stderr"] =""`, otherwise it would mean the test passes even if you got the regex wrong.

File: test/ntp_client_test.rb [hunk 2]
```diff
@@ -442,6 +442,17 @@
           expect(subject.sntp_test(server)).to eql(false)
         end
 
+        it "returns false if sntp respong no UCST" do
```
Comment (teclator) [inline test/ntp_client_test.rb]: just a typo and maybe reword to `it "returns false if sntp response includes 'no UCST'"`

File: src/modules/NtpClient.rb [hunk 3]
```diff
@@ -788,13 +788,17 @@ def sntp_test(server, ip_version = 4)
...
       # sntp returns always 0 if not called with option -S or -s (set system time)
       # so this is a workaround at least to return false in case server is not
-      # reachable. We could also take care of stdout checking if it includes
-      # "no (U|B)CST reponse", but it also implies be too dependent in the
-      # future and the ntp package should take care of it and aswer other exit
-      # code instead of 0
-      output["stderr"].include?("lookup error") ? false : output["exit"] == 0
+      # reachable.
+      return false if output["stderr"].include?("lookup error")
+      # this check is for valid address, but without ntp. If it break in future
```
Comment (teclator) [inline src/modules/NtpClient.rb]: If It breaks in the future

**Comment by teclator** [comment]
LGTM

**Comment by mvidner** [comment]
For the record, I think the commit messages of the form "fixes from code review" are in the same category as "fixes" and "improvements" or "stuff".
Anyway, this instance is harmless enough.
